nautobot/extras/views.py

```diff
@@ -1230,6 +1234,19 @@ def post(self, request, class_path=None, slug=None):
                     task_queue=job_form.cleaned_data.get("_task_queue", None),
                 )
 
+                return_url = request.POST.get("_return_url")
```

Comment:
Same comment as #4993 here regarding the change in behavior when a job run view fails

Reply:
Updated in both PRs. Thanks!